[flaky e2e tests] due to: error: timed out waiting for the condition on deployments/kubeflow-trainer-controller-manager

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://kubernetes.io/security/
-->


**What happened**:

failures on unreleated branches, eg https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/6876/pull-kueue-test-e2e-main-1-33/1968272284959379456

**What you expected to happen**:
no failures
**How to reproduce it (as minimally and precisely as possible)**:
ci
**Anything else we need to know?**:

```
secret/kubeflow-trainer-webhook-cert serverside-applied
service/kubeflow-trainer-controller-manager serverside-applied
deployment.apps/kubeflow-trainer-controller-manager serverside-applied
validatingwebhookconfiguration.admissionregistration.k8s.io/validator.trainer.kubeflow.org serverside-applied
error: timed out waiting for the condition on deployments/kubeflow-trainer-controller-manager
Switched to context "kind-kind".
Exporting logs for cluster "kind" to:
/logs/artifacts/run-test-e2e-singlecluster-1.33.4
No resources found in kueue-system namespace.
No resources found in default namespace.
Deleting cluster "kind" ...
Deleted nodes: ["kind-control-plane" "kind-worker" "kind-worker2"]
make: *** [Makefile-test.mk:135: run-test-e2e-singlecluster-1.33.4] Error 1
+ EXIT_VALUE=2
+ set +o xtrace
Cleaning up after docker in docker.
```

--- a/hack/e2e-common.sh
+++ b/hack/e2e-common.sh
@@ -313,7 +313,7 @@ function install_kubeflow_trainer {
         kubectl apply --kubeconfig="$2" --server-side -k "$manifests_temp_dir/overlays/manager"
     )
     # In order to install the training runtimes we need to wait for the ClusterTrainingRuntime webhook to be ready
-    kubectl wait --kubeconfig="$2" deploy/kubeflow-trainer-controller-manager -n kubeflow-system --for=condition=available --timeout=1m
+    kubectl wait --kubeconfig="$2" deploy/kubeflow-trainer-controller-manager -n kubeflow-system --for=condition=available --timeout=5m
     kubectl apply --kubeconfig="$2" --server-side -k "${KUBEFLOW_TRAINER_MANIFEST}/overlays/runtimes"
 }
 

